lib/HttpSignatureStrategy.js

```diff
@@ -11,7 +11,10 @@ var passport = require('passport');
 var util = require('util');
 var BedrockError = bedrock.util.BedrockError;
 var didio = require('did-io');
+var jsonld = require('jsonld');
 var URL = require('url');
+var request = require('request');
+request = request.defaults({json: true, strictSSL: false});
```

Comment:
should strictSSL just be a config option here, then? It seems like it would either always be on or off, because any arbitrary person can have us initiate a request to their publicKey and owner documents. @mattcollier